app/styles/components/_product-size.scss

```
@@ -38,3 +38,90 @@
         }
     }
 }
+
+.product-small {
+    .product__list--small {
+        @include d-grid(1fr, 117px 117px 117px 117px 117px 117px, 0);
```

```suggestion
        @include d-grid(1fr, repeat(6, 117px), 0);
```
Nhưng sao responsive nhọc thế.
Sao k dùng các đơn vị absolute như vh, vw để tăng giảm kích thước dựa trên kích thước màn hình.
Mà lặp lại 6 lần 117px như này nghĩa là có 6 item à? Nếu trong trường hợp bỏ đi hoặc thêm mới 1 sản phẩm thì lại phải đi chỉnh sửa style luôn à
Với cả kèm theo cái này chỉ là 1 chiều thì sao k sử dụng flex cho đơn giản.